sharding-core/sharding-core-optimize/src/main/java/org/apache/shardingsphere/core/optimize/sharding/segment/select/item/SelectItems.java

```diff
@@ -115,9 +120,18 @@ public boolean isUnqualifiedShorthandItem() {
     public List<String> getColumnLabels() {
         List<String> result = new ArrayList<>(items.size());
         for (SelectItem each : items) {
-            // TODO read * from metadata
             if (!(each instanceof ShorthandSelectItem)) {
                 result.add(each.getColumnLabel());
+            } else {
+                // get tableMetaData from ownerTableMetaDataMap by owner
```

Comment:
Please follow Contributor Covenant Code of Conduct[1]: `Include Javadoc, todo and fixme only in the comments.`

[1] https://shardingsphere.apache.org/community/en/contribute/code-conduct/